ec/src/lib.rs

```diff
@@ -313,6 +313,8 @@ pub fn prepare_g2<E: PairingEngine>(g: impl Into<E::G2Affine>) -> E::G2Prepared
 }
 
 /// A cycle of pairing-friendly elliptic curves.
+/// TODO: remove this once dependencies are
+///       updated with new curve cycle traits
```

Comment:
Maybe we can add a `#[deprecated("Please use `PairingFriendlyCycle` instead")]` marker here?